### PR TITLE
REPO-3743: Prevent the creation of built-in authorities, like user SYSTEM, GUEST

### PR DESCRIPTION
**Pull request by anechifor**
…STEM, GUEST.

### CODE REVIEW

**Comment by cristianturlica** [comment]
Test should be added in PersonTest and minor code formatting needed.

File: src/test/java/org/alfresco/repo/security/person/PersonTest.java [hunk 1]
```diff
@@ -1696,4 +1696,46 @@ public Void execute() throws Throwable
...
+            systemProps.put(ContentModel.PROP_LASTNAME, "myLastName");
+            systemProps.put(ContentModel.PROP_EMAIL, "myFirstName.myLastName@email.com");
+            systemProps.put(ContentModel.PROP_JOBTITLE, "myJobTitle");
+            systemProps.put(ContentModel.PROP_JOBTITLE, "myOrganisation");
+
+            personService.createPerson(systemProps);
+            fail("case fail creating SystemUserName: " + AuthenticationUtil.getSystemUserName());
+
+        }
+        catch (AlfrescoRuntimeException e)
```
Comment (cristianturlica) [inline src/test/java/org/alfresco/repo/security/person/PersonTest.java]: We could use junit rule ExpectedException.

File: src/test/java/org/alfresco/repo/security/person/PersonTest.java [hunk 2]
```diff
@@ -1696,4 +1696,46 @@ public Void execute() throws Throwable
...
+            guestProps.put(ContentModel.PROP_FIRSTNAME, "myFirstName");
+            guestProps.put(ContentModel.PROP_LASTNAME, "myLastName");
+            guestProps.put(ContentModel.PROP_EMAIL, "myFirstName.myLastName@email.com");
+            guestProps.put(ContentModel.PROP_JOBTITLE, "myJobTitle");
+            guestProps.put(ContentModel.PROP_JOBTITLE, "myOrganisation");
+
+            personService.createPerson(guestProps);
+            fail("case fail creating GuestUserName: " + AuthenticationUtil.getGuestUserName());
+        }
+        catch (AlfrescoRuntimeException e)
```
Comment (cristianturlica) [inline src/test/java/org/alfresco/repo/security/person/PersonTest.java]: We could use junit rule ExpectedException.

**Comment by ancutam** [comment]
Closing this, work continues in  #276.

**Pull request closed without merging**